usmqe_tests/api/grafana/test_cluster_dashboard.py

```diff
@@ -209,3 +210,68 @@ def test_hosts_panel_status(cluster_reuse):
         g_down == len(real_down),
         "Number of hosts that are down in graphite ({}) should be {}".format(
             g_down, len(real_down)))
+
+
+@pytest.mark.ansible_playbook_setup("test_setup.tendrl_services_stopped_on_nodes.yml")
+@pytest.mark.ansible_playbook_teardown("test_teardown.tendrl_services_stopped_on_nodes.yml")
+@pytest.mark.author("fbalak@redhat.com")
+def test_hosts(ansible_playbook, workload_stop_nodes, cluster_reuse):
```

Comment:
```suggestion
def test_hosts(cluster_reuse, ansible_playbook, workload_stop_nodes):
```

Please make sure that the cluster reuse fixture is executed before ansible playbook setup which is messing with tendrl.